answers.md

```diff
@@ -1 +1,18 @@
 # Answers Go In This File
+//Shaina 
+//Qustion 1
+1. Git is a tool used by software developers to manage their code. Git keeps track of code by saving existing code. Software developers use it to code with others by branching, merging commits, and adding their code to the main base. Git is a local repository.
+//Question 2
+2. GitHub is like a social media platform, or a version control system, for coders or software developers. It allows software developers to start repositories, fork other repositories, and request pulls. GitHub is a remote repository.
```

Comment:
Again, so many great elements here. I love that you compare Github to a social media platform and mention the ability to create or fork repositories. Three notes to be slightly more precise:
1. **Git** (not Github) is a version control system. Github is an online place to host and manage Git repositories. While you're editing the first answer, you might want to include the term "version control system" in your answer.
2. We call them "Pull requests" not "request pulls". You could say "It allows software developers to start repositories, fork other repositories, and manage pull requests".
3. Github it isn't a remote repository itself. Instead, you could say that it "hosts remote repositories".